pkg/resources/resources.go

```diff
@@ -933,6 +933,8 @@ const (
 	UserClusterWebhookDeploymentName        = "usercluster-webhook"
 	UserClusterWebhookServiceName           = "usercluster-webhook"
 	UserClusterWebhookServingCertSecretName = "usercluster-webhook-serving-cert"
+	UserClusterWebhookSeedListenPort        = 443
```

Comment:
Can we use non-privileged ports (i.e. >=1024)? That would not require devs to be root when testing the webhook locally. Maybe 6443 and 7443?